.travis.yml

```diff
@@ -55,12 +57,16 @@ matrix:
       env: setup=lowest
     - php: 7.0
       env: setup=stable
+    - php: 7.0
+      env: PHPSTAN=1
     - php: 7.1
       env: setup=lowest
     - php: 7.1
       env:
         - coverage=true
         - setup=stable
+    - php: 7.1
+      env: PHPSTAN=1
```

Comment:
Can you use `phpstan=true` (lowercase, pseudo boolean) instead - just to keep consistent ? ;-)